hw04/BarbieWalletBalance.cs

```diff
@@ -14,21 +14,66 @@ public class BarbieBank : MonoBehaviour
     Have the function return a new generic array with the correct parameters. 
     */
 
+    //answer:
     void Start()
     {
+        //question 2
+        BarbieWorld<string> barbiehat = new BarbieWorld<string>();
+
         int[] biggerWallet = BarbieWallet(500, 600);
         Debug.Log(biggerWallet.Length + " " + biggerWallet[0] + " " + biggerWallet[1]);
 
         GetMoney(500, 600);
+
+        //Testing: BarbieBank
+        int[] barbiebank=G_BarbieBank<int>(100, 200, 300);
+    }
+
+    private int[] BarbieWallet(int money1, int money2)
```

Comment:
This problem is very close. Needs an open and close with `T`instead of with `int`.

```
    // Generic function for BarbieWallet
    private T[] BarbieWallet<T>(T money1, T money2)
    {
        return new T[] { money1, money2 };
    }
```